case-lib/lib.sh

```diff
@@ -86,6 +86,14 @@ func_lib_disable_pulseaudio()
     # because hijack the sudo command
     sudo "sed -i '/autospawn/cautospawn = no' $PULSEAUDIO_CONFIG"
     sudo pkill -9 pulseaudio
+    sleep 1 # wait pulseaudio to be disabled
+    if [ ! "$(ps -C pulseaudio --no-header)" ]; then
+        echo "Pulseaudio disabled"
+        return 0
+    else
+        echo "Failed to disable pulseaudio"
+        return 1
+    fi
```

Comment:
what try to do?  
when `kill -9 ` failed, it is the big problem,  
but here also do nothing.

Reply:
just try to output some log for debug. if pulseaudio killed, we will know it, and if pulseaudio kill failed, we also know it. not get it why this code cause big problem.